Manual/Releases/v4.19.0.lean

```diff
@@ -495,78 +355,326 @@ For this release, 420 changes landed. In addition to the 164 feature additions a
   class CharP (R : Type u) [AddMonoidWithOne R] (p : Nat) : Prop where
     cast_eq_zero_iff (R p) : ∀ x : Nat, (x : R) = 0 ↔ p ∣ x
 
+  #guard_msgs in #check CharP.cast_eq_zero_iff
+  /-
+  info: CharP.cast_eq_zero_iff.{u} (R : Type u) {inst✝ : AddMonoidWithOne R} (p : Nat) [self : CharP R p] (x : Nat) :
+    ↑x = 0 ↔ p ∣ x
+  -/
+  ```
+
 * [#7746](https://github.com/leanprover/lean4/pull/7746) adds declaration ranges to structure fields that were copied
   from parents that aren't represented as subobjects, supporting "go to
   definition". The declaration range is the parent in the `extends`
   clause.
 
-* [#7760](https://github.com/leanprover/lean4/pull/7760) ensures `grind` is using the default transparency setting when
-  computing auxiliary congruence lemmas.
+### Parallel Elaboration
 
-* [#7765](https://github.com/leanprover/lean4/pull/7765) improves how `grind` normalizes dependent implications during
-  introduction.
-  Previously, `grind` would introduce a hypothesis `h : p` for a goal of
-  the form `.. ⊢ (h : p) → q h`, and then normalize and assert a
-  non-dependent copy of `p`. As a result, the local context would contain
-  both `h : p` and a separate `h' : p'`, where `p'` is the normal form of
-  `p`. Moreover, `q` would still depend on the original `h`.
+* [#7084](https://github.com/leanprover/lean4/pull/7084) enables the elaboration of theorem bodies, i.e. proofs, to
+  happen in parallel to each other as well as to other elaboration tasks.
```

Comment:
This is also a big deal, maybe second highlight?

Reply:
Sure, added in [f06e06b](https://github.com/leanprover/reference-manual/pull/442/commits/f06e06b1f07fb65be0087d69afe31487ea6d01ee)